adapters/net_http/lib/opentelemetry/adapters/net/http/patches/instrumentation.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Net
+      module HTTP
+        module Patches
+          # Module to prepend to Net::HTTP for instrumentation
+          module Instrumentation
+            HTTP_METHODS_TO_SPAN_NAMES = Hash.new { |h, k| h[k] = "HTTP #{k}" }
+            USE_SSL_TO_SCHEME = { false => 'http', true => 'https' }.freeze
+
+            def request(req, body = nil, &block) # rubocop:disable Metrics/MethodLength
```

Comment:
Rubocop complains that we are unnecessarily disabling `Metrics/MethodLength` when I run tests. Can you confirm this is the case for you, and remove the `rubocop:disable` if so?